cypher/cypher-docs/src/docs/dev/ql/load-csv/index.asciidoc

```diff
@@ -87,8 +87,6 @@ include::import-data-from-a-csv-file-with-a-custom-field-delimiter.asciidoc[]
 
 include::importing-large-amounts-of-data.asciidoc[]
 
-include::setting-the-rate-of-periodic-commits.asciidoc[]
```

Comment:
Is this just gone or should we import a replacement here?

Reply:
Oh, nice catch. This should be replaced, thank you.